app/controllers/mixins/ems_common_angular.rb

```diff
@@ -431,6 +431,7 @@ def build_credentials(ems, mode)
          ems.supports_authentication?(:bearer) && !params[:default_password].blank?
         creds[:hawkular] = {:auth_key => params[:default_password], :userid => "_"}
         creds[:bearer] = {:auth_key => params[:default_password]}
+        creds.delete(:default)
```

Comment:
> it also lends itself to breaking UI provider agnosticism

`ems_common` is the opposite of *provider agnostic* :grin: 

@juliancheal is tackling this. And until we have a clean way to support stuff like this on the provider side, I am ok with adding more provider specifics in here, because its just not reasonably possible in another way :unamused: